app/webpacker/components/RegistrationsV2/RegistrationAdministration/RegistrationAdministrationContainer.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { useQuery } from '@tanstack/react-query';
+import { showMessage } from '../Register/RegistrationMessage';
+import { useDispatch } from '../../../lib/providers/StoreProvider';
+import getCompetitionInfo from '../api/competition/get_competition_info';
+import RegistrationAdministrationList from './RegistrationAdministrationList';
+import Loading from '../../Requests/Loading';
+
+export default function RegistrationAdministrationContainer({ competitionId }) {
+  const dispatchStore = useDispatch();
+
+  const fetchCompetitionInfo = async () => getCompetitionInfo(competitionId);
+
+  const {
+    isLoading: isCompetitionInfoLoading,
+    data: competitionInfo,
+    refetch: refetchCompetitionInfo,
+  } = useQuery({
+    queryKey: ['competitionInfo', 'test'],
```

Comment:
`'test'` still contained in your reviewable code.
This should contain `competitionId` instead.